src/modules/tree.ts

```diff
@@ -137,6 +137,18 @@ export default {
         type: 'boolean',
       },
     ],
+    typings: {
+      description: 'Download react and graphql app typings',
```

Comment:
can we use the "correct" casing here?

```suggestion
      description: 'Download React and GraphQL app typings',
```